borsh/tests/test_simple_structs.rs

```diff
@@ -107,6 +118,7 @@ fn test_deserialize_invalid_discriminant() {
     from_slice::<X>(&[2]).unwrap();
 }
 
+#[cfg(any(feature = "hashbrown", feature = "std"))]
```

Comment:
Well, this is the most important test in this file, let's extract hash maps testing into `test_hash_map` if there is anything useful to be extracted, and drop `cfg` guards in this file.